protocol-jackson/src/main/java/io/camunda/zeebe/protocol/jackson/ZeebeProtocolModule.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.protocol.jackson;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import io.camunda.zeebe.protocol.record.ImmutableRecord;
+import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.protocol.util.ProtocolTypeMapping;
+import io.camunda.zeebe.protocol.util.ProtocolTypeMapping.Mapping;
+import java.util.Objects;
+
+/**
+ * A Jackson module which enables your {@link ObjectMapper} to serialize and deserialize Zeebe
+ * protocol objects, e.g. {@link Record}, {@link
+ * io.camunda.zeebe.protocol.record.value.ErrorRecordValue}, etc., essentially anything annotated
+ * with {@link io.camunda.zeebe.protocol.record.ImmutableProtocol}, out of the box.
+ *
+ * <p>To use, simply create an {@link ObjectMapper} as your normally would, and add this module.
+ *
+ * <h2>Usage
```

Comment:
💭 I don't think I've ever seen an H2 in javadoc before. I have no opinion on it, but I noticed the usage.